README.md

```diff
@@ -87,6 +87,20 @@ cio.destroy(1);
 
 ---
 
+### Merge Customers
+
+When you merge two people, you pick a primary person and merge a secondary, duplicate person into it. The primary person remains after the merge and the secondary is deleted. This process is permanent: you cannot recover the secondary person.
+
+The first and third parameters represent the identifier for the primary and secondary people respectively—one of `id`, `email`, or `cio_id`. The second and fourth parameters are the identifier values for the primary and secondary people respectively.
+
+```javascript
+// cio.mergeCustomers("primaryType", "primaryIdentifier", "secondaryType", "secondaryIdentifier")
```

Comment:
We have a different way to document arguments for functions, so to keep it consistent, this should be updated to match the existing style. Here's an example: https://github.com/customerio/customerio-node/#options-2

Reply:
@glosier Hey, Would you please take a look at this?

Reply:
Done